Core/Models/PointMassSecondOrder.hpp

```diff
@@ -14,28 +14,34 @@ namespace gtfo{
     {
         Scalar mass;
         Scalar damping;
+        Scalar stiffness;
+        Scalar virtual_initial_position;
 
         SecondOrderParameters()
-            : ParametersBase<Scalar>(), mass(1.0), damping(1.0)
+            : ParametersBase<Scalar>(), mass(1.0), damping(1.0), stiffness(1.0), virtual_initial_position(1.0)
         {
         }
 
-        SecondOrderParameters(const Scalar &dt, const Scalar &mass, const Scalar &damping)
-            : ParametersBase<Scalar>(dt), mass(mass), damping(damping)
+        SecondOrderParameters(const Scalar &dt, const Scalar &mass, const Scalar &damping, const Scalar &stiffness, const Scalar &virtual_initial_position)
+            : ParametersBase<Scalar>(dt), mass(mass), damping(damping), stiffness(stiffness), virtual_initial_position(virtual_initial_position)
         {
-            assert(mass > 0.0 && damping > 0.0);
+            assert(mass > 0.0 && damping > 0.0 && stiffness >= 0.0 && virtual_initial_position >= -90 && virtual_initial_position <= 90);
```

Comment:
Since this model represents an abstract / generic second order model, the limits on `virtual_initial_position` do not make sense. So, I don't think there should be limits there.